static/usage/v7/item/inputs/vue.md

```diff
@@ -44,23 +41,31 @@
   </ion-item>
 
   <ion-item>
-    <ion-label>Toggle</ion-label>
-    <ion-toggle slot="end"></ion-toggle>
+    <ion-toggle> Toggle </ion-toggle>
```

Comment:
Why do toggle and checkbox not use `label`? I tried reading through the design documents and didn't see it mentioned.

Reply:
Toggle and Checkbox have shadow dom, so we can pass the label as a slot. Input/textarea have a prop since they don't use shadow dom. Using a slot lets us also support custom HTML

Reply:
At the moment it is not, but feel free to propose a design doc revision.